### DISCUSSION TITLE
Added new repository for lmds and analysis CI tests

### DISCUSSION THREAD

**Discussion by jose-luis-rs** [R3BRootGroup/R3BRoot#1103] [Ideas]
@inkdot7: In a world with copyright law, a lack of license terms will negate GNU freedom two (redistribution) and freedom three (redistribution of modified versions). Technically, running (freedom 0) and modifying (freedom 1) may also be negated but I see no practical way for the copyright holder to enforce that. 

I would much prefer if we could stick licence terms on upexps, be they GPL, BSD, or public domain. To my knowledge the reason we don't just do that are copyright concerns.

I see four ways forward:
(0) We figure out who all the contributors were and get them to okay distribution under GPL.
(1) We do a clean room reimplementation of upexps: someone documents the data formats from our spec files (which are not copyrightable), then someone else reimplements those those in ucesb. This is of course a waste of time, but we could just stick the GPL on the result. 
(2) We release the source as-is without licence, passing the buck to the users. 
(3) We do not release anything.

Any external user would prefer 0 or 1 over the other options, obviously. 

But I would argue that option two is strictly more useful than our current choice, which is option 3. If nothing else, it would enable external users to do the clean room reimplementation themselves (if they want to use our spec files as the basis for a larger project where they need legal certainty).

_Originally posted by @klenze in https://github.com/R3BRootGroup/R3BRoot/issues/862#issuecomment-1714000587_

**Comment by klenze** [comment]
I think Haik is the person who should decide how we can release the unpacker source code. I think he sometimes lurks here. Perhaps he can comment in person next week.

**Comment by jose-luis-rs** [comment]
No problem, this is a decision for the DAQ-WG. At the moment it is running on ucesb directly, just to check that we can load R3BUcesbSource and that it runs properly.
